pkg/config/config.go

```diff
@@ -396,6 +396,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.run_path", defaultRunPath)
 	config.BindEnv("logs_config.dd_url")
 	config.BindEnvAndSetDefault("logs_config.use_http", false)
+	config.BindEnvAndSetDefault("logs_config.linger_time_in_s", 5)
```

Comment:
nits: `linger_time` is good but I like `batch_period` better, WDYT ?

Reply:
I was keeping the same naming that Kafka, but here again no strong opinion

Reply:
Let's ask @NBParis

Reply:
in other tool it's called `batch_wait` which kind of represent well the fact that we will wait at most this time.

so it would give `logs_config.http.batch_wait`